includes/polling/os/awplus.inc.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+//OS info - https://www.alliedtelesis.com/products/software/AlliedWare-Plus
```

Comment:
You can remove these comments.

Reply:
Do you mean the next line specifying OS datasheet also? 

Is it not desirable to have any comments about the OS in the config file?

Reply:
You've got them already in the discovery file. Don't see a need to duplicate them.

Reply:
Ah yes - True...